public/build/bitbag/elasticsearch/admin/bitbag-elasticsearch-admin.css

```diff
@@ -0,0 +1 @@
+
```

Comment:
there shouldnt be ANY files from root/public directory in repo, please remove them all